### DISCUSSION TITLE
Login Help

### DISCUSSION THREAD

**Discussion by tjameswhite** [mastodon/mastodon#23192] [Q&A]
Created a Mastadon account. I am logged in and posting on the app, but I can not access the account itself (setting via the app or online). I have the welcome emails, so I know the mail address is correct. I have tried to 'reset password' but have never received any response. Yes I have checked spam, trash etc. 

How do I get some help resetting a password? 

Account is on mstdn.social

**Comment by mtbluedog** [comment]
That has to do with the Mastodon server at mstdn.social, not Mastodon software. Try https://hub.mstdn.social/contact/
